utils/check_repo.py

```diff
@@ -67,6 +67,7 @@
     "xlm_prophetnet": "xlmprophetnet.rst",
     "xlm_roberta": "xlmroberta.rst",
     "bert_generation": "bertgeneration.rst",
+    "marian": "marian.rst",
```

Comment:
Is this necessary?

Reply:
Shoudn't be

Reply:
I get 
```python
Exception: There were 1 failures:
transformers.modeling_tf_marian does not have its corresponding doc file marian.rst. If the doc file exists but isn't named marian.rst, update `MODEL_NAME_TO_DOC_FILE` in the file `utils/check_repo.py`.
```
without. Will investigate.

Reply:
Don't spend too much time if it's my script giving you a hard time. I can investigate after this PR is merged.